legend-engine-language-pure-dsl-service/src/main/java/org/finos/legend/engine/language/pure/dsl/service/grammar/to/ServiceGrammarComposerExtension.java

```diff
@@ -86,6 +86,16 @@ private static String renderService(Service service, PureGrammarComposerContext
         }
         serviceBuilder.append(getTabString()).append("test: ");
         serviceBuilder.append(HelperServiceGrammarComposer.renderServiceTest(service.test, context));
+        if (service.tags != null && !service.tags.isEmpty())
+        {
+            serviceBuilder.append(getTabString()).append("tags:\n").append(getTabString()).append("[\n")
+                    .append(LazyIterate.collect(service.tags, t -> getTabString(2) +
+                    ("{\n") +
```

Comment:
Why is this in a new line? Lets make it more readable

Reply:
fixed